pkg/engine/loadbalancers.go

```diff
@@ -236,6 +237,93 @@ func CreateLoadBalancer(prop *api.Properties, isVMSS bool) LoadBalancerARM {
 	return loadBalancer
 }
 
+func createOutboundRules(prop *api.Properties) *[]network.OutboundRule {
+	currentVersion, _ := semver.Make(prop.OrchestratorProfile.OrchestratorVersion)
+	min13Version, _ := semver.Make("1.13.7")
+	min14Version, _ := semver.Make("1.14.3")
+	min15Version, _ := semver.Make("1.15.0")
+
+	if currentVersion.LT(min13Version) ||
```

Comment:
// TODO generalize a library for doing this kind of semver foo that will always be a part of maintaining k8s functional behaviors :)